save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/TopBar.kt

```diff
@@ -76,6 +76,7 @@ fun topBar() = fc<TopBarProps> { props ->
                     }
                 }
                 location.pathname
+                    .substringBeforeLast("?")
```

Comment:
what's for?

Reply:
Previously query string would be displayed in the `TopBar` along with the rest of the URL. Now it'll be trimmed.